application/Hand.py

```diff
@@ -1,4 +1,30 @@
-class Hand: 
-  def __init__(self, cards, startingPlayer):
-    hand = cards
-    player = startingPlayer
+import psycopg2
+import random
+import application.Card as Card
+
+# Probably better to get rid of the hand class once there is a better place for createHands()
+class Hand:
```

Comment:
The actual db connection is handled by SQL alchemy so we can directly query in the method of the class. It don't see it getting too complicated but if it does we can create a service which handles this sort of thing